README.md

```diff
@@ -56,14 +56,16 @@ expressOasGenerator.init(
   function(spec) { return spec; },
   'path/to/a/file/filename.json',
   60 * 1000,
-  ['User','Student']
+  ['User','Student'],
+  ['users', 'students']
```

Comment:
These parameters are getting really convoluted and confusing:D Any plans to deprecate the `.init` function & release `v2.0`, @mpashkovskiy?

Reply:
Hey @maxiejbe thanks for implementation!
As of parameters of `init`/`handleResponses `: is it possible to use the mongoose model names as tags? For example in the code above those arrays are identical. That will also reduce the number of params.